Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -598,7 +602,9 @@ CharacterRange AXTextMarker::characterRangeForLine(unsigned lineIndex) const
 AXTextMarkerRange AXTextMarker::markerRangeForLineIndex(unsigned lineIndex) const
 {
     // This implementation doesn't respect the offset as the only known callsite hardcodes zero. We'll need to make changes to support this if a usecase arrives for it.
-    TEXT_MARKER_ASSERT(!offset());
+    TEXT_MARKER_ASSERT(!offset(), "markerRangeForLineIndex");
+    if (offset())
+        return { };
```

Comment:
I think it may be a better / closer-to-correct fallback to just continue on with the method as normal, rather than returning { } early.